utils/python/CIME/test_status.py

```diff
@@ -241,7 +242,10 @@ def get_overall_test_status(self, wait_for_run=False, check_throughput=False, ch
                         rv = NAMELIST_FAIL_STATUS
 
                 elif (rv in [NAMELIST_FAIL_STATUS, TEST_PASS_STATUS] and phase.startswith(COMPARE_PHASE)):
-                    rv = TEST_DIFF_STATUS
+                    if (phase.startswith(COMPARE_BASELINE_PHASE) and TEST_BFAIL_STATUS in comment):
```

Comment:
I'm not sure about this one. I thought we agreed not to add a new state to the system?